cq-component-annotations/src/main/java/com/citytechinc/cq/component/annotations/widgets/DateField.java

```diff
@@ -70,7 +70,7 @@
 	 *
 	 * @return String
 	 */
-	String storedFormat() default "YYYY-MM-DD";
+	String storedFormat() default "YYYY-MM-DDThh:mm:ss.SSSZ";
```

Comment:
according to /libs/granite/ui/components/foundation/form/datepicker/render.jsp the default is actually "YYYY-MM-DD[T]HH:mm:ss.000Z"